example/lambdas/versionUpTest/original.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/**
+ * calls callback with static key/text payload object.  Used for testing lambda modification.
+ * @param {Object} event - AWS event
+ * @param {Object} context - an AWS Lambda context
+ * @param {Function} callback - an AWS Lambda handler
+ * @returns {undefined} - does not return a value
+ */
+
+function handler(event, context, callback) {
+  const eventCopy = event;
+  eventCopy.payload = { output: 'Current Version' };
```

Comment:
What is the point of `eventCopy` here? `const eventCopy = event` means when changes are made to `eventCopy` they occur in `event` as well.

Reply:
Absolutely none, and there's no point in cloning that object.     I'm assuming this got missed in a refactor.  I'll clean it up.

Reply:
This was actually to satisfy https://eslint.org/docs/rules/no-param-reassign.        I'll change the variable name.  In this instance brevity should probably prevail over copying the test object: 4eeabe84